pyrefinebio/computed_file.py

```diff
@@ -13,18 +13,18 @@ class ComputedFile:
 
     ComputedFiles are representation of files created by data-refinery processes.
 
-    get a computed file based on id
+    Retrieve a computed file based on id
 
         ex:
         >>> import pyrefinebio
         >>> id = 1
-        >>> sample = pyrefinebio.ComputedFile.get(id)
+        >>> file = pyrefinebio.ComputedFile.get(id)
```

Comment:
`file` is a reserved word in python, which bugs me.
```suggestion
        >>> computed_file = pyrefinebio.ComputedFile.get(id)
```

Reply:
I just found out that's no longer true in python 3!!! Rejoice!